utils/prometheus-node-exporter-lua/files/usr/lib/lua/prometheus-collectors/dhcp_leases.lua

```diff
@@ -0,0 +1,18 @@
+local function scrape()
+  metric_dhcp_lease = metric("dhcp_lease", "gauge")
+  for e in io.lines("/tmp/dhcp.leases") do
```

Comment:
Is it specific to dnsmasq ? I know odhcpd leases are accessible via ubus, so maybe we should change the collector name (metric name is fine I think)

Reply:
It is indeed specific to dnsmasq.

I also found that's configurable in uci, so I'll try to read it from there.

```
# ubus -v call uci get '{"config":"dhcp","type":"dnsmasq"}'
{
        "values": {
                "cfg01411c": {
                        ".anonymous": true,
                        ".type": "dnsmasq",
                        ".name": "cfg01411c",
                        ".index": 0,
                        [..]
                        "leasefile": "/tmp/dhcp.leases",
                        [..]
                }
        }
}
```

Do you know if it's possible to have more than one dnsmasq instance running? Should I add all of them in the metrics with their name as label?